diff -uraN zsh-5.8/Src/Modules/curses_keys.awk omv-zsh-5.8/Src/Modules/curses_keys.awk
--- zsh-5.8/Src/Modules/curses_keys.awk	2017-12-04 15:09:36.000000000 +0100
+++ omv-zsh-5.8/Src/Modules/curses_keys.awk	2020-12-03 15:22:52.967644587 +0100
@@ -12,8 +12,13 @@
 
 END {
     printf("static const struct zcurses_namenumberpair keypad_names[] = {\n")
-    for (i = 0; i < 0 + nkeydefs; i++)
+   for (i = 0; i < 0 + nkeydefs; i++) {
+        if (name[i] == "EVENT")
+            printf("#ifdef KEY_EVENT\n")
         printf("    {\"%s\", KEY_%s},\n", name[i], name[i])
+        if (name[i] == "EVENT")
+            printf("#endif\n")
+    }
     printf("    {NULL, 0}\n")
     printf("};\n")
 }
