--- zsh-5.9.2/docroot/Zsh_Guide/zshguide/zshguide.yo.omv~
+++ zsh-5.9.2/docroot/Zsh_Guide/zshguide/zshguide.yo
@@ -48,10 +48,10 @@ def(var)(1)(em(ARG1))\
 COMMENT(-- XXparcounter is not defined in yodl 2.x --)\
 IFDEF(XXparcounter)(\
 )(\
-    NEWCOUNTER(XXparcounter)\
+    DEFINECOUNTER(XXparcounter)(0)\
 )\
 COMMENT(-- without excess newlines in latex style --)\
-UNDEFINEMACRO(verb)\
+DELETEMACRO(verb)\
 DEFINEMACRO(verb)(1)(\
     whenlatex(PUSHCHARTABLE()\
               latexcommand(\begin{verbatim}) \
